### ISSUE TITLE
[Adaptive Cards - Sample - Inputs with validation] - Luminosity ratio of 'mm', etc., placeholder text for the 'Due Date' and 'Start time' edit field on receiving keyboard focus with its adjacent background color is 4.49:1, is less than 4.5:1.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9321]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: 25H2 (OS Build 26200.5761)
Browser Version: Version 145.0.3734.1 (Official build) dev (64-bit)
Tool: Color Contrast analyzer -> Launch through Accessibility Insights for Windows Tool

**Repro Steps:**

1. Open the above URL on the Edge Dev browser.
2. Navigate to the 'Samples' control present on the header section of the page using tab key and activate using enter key.
3. Navigate to the 'Inputs with validation' control present on the left navigation using tab key and activate using enter key.
4. Navigate to the 'mm', 'dd', 'yyyy' and '--' placeholder text present for the 'Due Date' and 'Start time' edit field using tab key.
5. Launch Accessibility Insights for Windows tool and open Color contrast analyzer and select the foreground color of the 'mm', 'dd', 'yyyy' and '--' placeholder text on receiving keyboard focus with respect its adjacent background color.
6. Verify that, luminosity ratio of 'mm', 'dd', 'yyyy' and '--' placeholder text present for the 'Due Date' and 'Start time' edit field on receiving keyboard focus with respect to its adjacent background color is less than minimum required ratio i.e., 4.5:1 or not.

**Actual Result:**
Luminosity ratio of 'mm', 'dd', 'yyyy' and '--' placeholder text present for the 'Due Date' and 'Start time' edit field on receiving keyboard focus with respect to its adjacent background color is 4.49:1, which is less than minimum required ratio i.e., 4.5:1.

**Expected Result:**
Luminosity ratio of 'mm', 'dd', 'yyyy' and '--' placeholder text present for the 'Due Date' and 'Start time' edit field on receiving keyboard focus with respect to its adjacent background color should be greater than or equal to the minimum required ratio i.e., 4.5:1.

**User Experience:**
Low vision and color vision deficiency users relying on high contrast themes will not be able to identify/read the text on receiving keyboard focus because of its low contrast ratio with its adjacent background color and due to this user might miss the information conveyed by the meaningful texts and not able to access the control properly.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Image](https://github.com/user-attachments/assets/567d7f53-059b-4ab8-8bd3-d1310a96098d)

### Card JSON

```JSON
NA
```
